tests/integration/reqs/test_submit_only.py

```diff
@@ -7,23 +7,21 @@
 from xrpl.models.requests import SubmitOnly
 from xrpl.models.transactions import OfferCreate
 
-TX = OfferCreate(
-    account=WALLET.classic_address,
-    sequence=WALLET.sequence,
-    last_ledger_sequence=WALLET.sequence + 10,
-    taker_gets="13100000",
-    taker_pays=IssuedCurrencyAmount(
-        currency="USD",
-        issuer=WALLET.classic_address,
-        value="10",
-    ),
-)
-
 
 class TestSubmitOnly(IntegrationTestCase):
     @test_async_and_sync(globals(), ["xrpl.transaction.autofill_and_sign"])
     async def test_basic_functionality(self, client):
+        TX = OfferCreate(
+            account=WALLET.classic_address,
+            taker_gets="13100000",
+            taker_pays=IssuedCurrencyAmount(
+                currency="USD",
+                issuer=WALLET.classic_address,
+                value="10",
+            ),
+        )
```

Comment:
This transaction can still be outside the class - it'll just be autofilled and signed inside the test.

Reply:
Sidenote that doesn't need to be handled right now: this test should really use a simpler transaction with no side effects, like an AccountSet that changes a flag or something